pints/_mcmc/__init__.py

```diff
@@ -1102,3 +1102,76 @@ def mcmc_sample(log_pdf, chains, x0, sigma0=None, method=None):
     """
     return MCMCController(    # pragma: no cover
         log_pdf, chains, x0, sigma0, method=method).run()
+
+
+def initialise_finite(log_pdf, chains, random_sampler=None, max_n_tries=50,
+                      parallel=False, n_workers=None):
+    """
+    Draws parameter values from a given sampling distribution until either
+    finite values for each of ``chains`` have been generated or the number of
+    attempts exceeds ``num_tries``.
```

Comment:
Can we change this to "the number of attempts per chain", so that we can simplify the logic below?

Reply:
Done.